kms/quickstart.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample quickstart is a basic program that uses Cloud KMS.
+package kms
```

Comment:
Should this be a main package? Weird to have `func main` outside of `package main`.
```suggestion
package main
```